Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/conflict.py

```diff
@@ -52,11 +90,11 @@ def main(cls, args, repository, **kwargs):
             sys.stderr.write('Cannot checkout conflict, must be in a local git repository\n')
             return 1
 
-        expected_branch = 'integration/conflict/{}'.format(args.radar_id)
+        radar_id = int(cls.get_radar_client().extract_radar_ids_from_text(args.radar)[0])
```

Comment:
If you constructed `radar_obj = Tracker.from_string(f'rdar://{args.radar}')` in main, you can extract the radar id as `radar_obj.id` and avoid the call to radar client! We can remove both `get_radar_client ` and `radar_for_id `.

Reply:
great point, updated!